test/test_bulk_renewal.js

```diff
@@ -16,91 +17,92 @@ const ETH_LABEL = sha3('eth');
 const ETH_NAMEHASH = namehash.hash('eth');
 
 const advanceTime = Promise.promisify(function(delay, done) {
```

Comment:
We have an entire toolset for this function please use this.

Reply:
None of this is changing in the current PR. Please don't gate changes on fixing everything wrong or out of date in every file it touches.